packages/libs/react-ui/src/components/Accordion/Accordion.stories.tsx

```diff
@@ -1,59 +1,80 @@
-import { Accordion, IAccordionProps } from './Accordion';
+import type { IAccordionProps, IAccordionSectionProps } from './';
+import { Accordion } from './';
 
 import type { Meta, StoryObj } from '@storybook/react';
 import React from 'react';
 
-const meta: Meta<{} & IAccordionProps> = {
+const generateSection = (i: number): IAccordionSectionProps => ({
+  title: `Section ${i}`,
+  children: <p>This is the content for section {i}</p>,
+  onOpen: () => console.log(`open section ${i}`),
+  onClose: () => console.log(`close section ${i}`),
+});
+const generateSections = (n: number): IAccordionSectionProps[] =>
+  Array.from({ length: n }, (d, i) => generateSection(i + 1));
+
+const sampleSections: IAccordionSectionProps[] = generateSections(5);
+
+type StoryProps = { sectionCount: number; linked: boolean } & IAccordionProps;
+
+const meta: Meta<StoryProps> = {
   title: 'Components/Accordion',
+  parameters: {
+    controls: {
+      hideNoControlsWarning: true,
+      sort: 'requiredFirst',
+    },
+    docs: {
+      description: {
+        component:
+          'Note: optional property of `openSection` may be set to the Root component to set the initially opened section. This defaults to -1 and has only been explcitly set in the story code for demonstration purposes.',
+      },
```

Comment:
why set to this -1 by default rather than undefined? Also, based on conventions this should be prefixed with `initial`. So `initialOpenSection` or something similar